zen/providers/offline.py

```
@@ -129,107 +131,94 @@ class OllamaProvider:
+
     def _check_availability(self) -> bool:
         """Check if Ollama is installed and running."""
         try:
             # Check if ollama binary exists
-            result = subprocess.run(
-                ["which", "ollama"],
-                capture_output=True,
-                text=True,
-                timeout=5
-            )
+            result = subprocess.run(["which", "ollama"], capture_output=True, text=True, timeout=5)
```

Ruff (0.15.6)

[error] Starting a process with a partial executable path

(S607)